extension/public/background.ts

```diff
@@ -1,20 +1,5 @@
 import { initMessageListener } from "../src/background";
 
-chrome.runtime.onInstalled.addListener(() => {
-  chrome.declarativeContent.onPageChanged.removeRules(undefined, () => {
-    chrome.declarativeContent.onPageChanged.addRules([
-      {
-        conditions: [
-          new chrome.declarativeContent.PageStateMatcher({
-            css: ["div"],
-          }),
-        ],
-        actions: [new chrome.declarativeContent.ShowPageAction()],
-      },
-    ]);
-  });
-});
-
```

Comment:
this is some extension boilerplate - we weren't using this, but we will need this `onInstalled` hook for this ticket: https://app.asana.com/0/1168666457741233/1187237204678441